### PR TITLE
B'More Stunners: Over and Out

### PR DESCRIPTION
**Pull request by a-jean**
B'More Stunner's last blog post!

### CODE REVIEW

File: blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: B'More Stunners&#58; B'More Stunners&#58 Over and Out!
```
Comment (alicetragedy) [inline blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md]: Could you remove your team name from the title once? It's in here twice.

File: blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+permalink: blog/2016-10-16-B-More-Stunners-Over-and-Out
+current: blog
+author: Oreoluwa Akinsanmi and Ashley Jean
+twitter: ashhjean 
+categories:
+- student-posts
+---
+
+
+![teamphoto](/img/blog/2016/ashley.jpeg)
```
Comment (alicetragedy) [inline blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md:14]: All images need image credits, as clearly explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). It's enough to put down your team name if you took the picture — the name of the person who took the photo is ok too.

File: blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Our sincere thank you goes out to our amazing coaches and mentors ( Benedikt 
+André , Lynn, Jamie, Mira and Beverly)  who dedicated their time this summer to help us on this journey. We could not have done it without your ongoing support, mentorship and pep talks! 
+
+
+Lastly, We would like to thank the wonderful organizers of Rails Girls Summer Of Code. Thank you for making this fellowship possible and giving us the opportunity to be a part of it! Because of this program we have grown as developers and individuals. 
+
+
+And for that we are eternally grateful <3 
+
+![Dance](/img/blog/2016/dino.gif)
```
Comment (alicetragedy) [inline blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md:86]: please add an image credit to this (e.g. source of the gif if you found it on the internet)

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Good and concise post, but it's still missing a couple of things (I've made comments accordingly). The most important thing would be to add a paragraph or two about the conference itself, since there is nothing in the post about it. Please let me know if you have any questions!

**Comment by a-jean** [comment]
Hi @alicetragedy !
I've made the changes you've requested. Let me know if there is anything else that needs to be fixed.

Thanks!

**Pull request merged**